src/main/java/io/anserini/kg/freebase/Topic.java

```diff
@@ -0,0 +1,169 @@
+package io.anserini.kg.freebase;
+
+import org.openrdf.model.Literal;
+import org.openrdf.model.ValueFactory;
+import org.openrdf.model.impl.SimpleValueFactory;
+import org.openrdf.rio.ntriples.NTriplesUtil;
+
+/**
+ * A document that represent an freebase topic
+ */
+public class Topic {
+
+  /**
+   * Splitter that describes how s,p,o are split in a triple line
+   */
+  public static final String TRIPLE_SPLITTER = "\t";
+
+  /**
+   * Predicates for which the literals should be stored
+   */
+  public static final String WIKI_EN_URI = "http://rdf.freebase.com/key/wikipedia.en";
+  public static final String WIKI_EN_TILE_URI = "http://rdf.freebase.com/key/wikipedia.en_title";
```

Comment:
The `if` statements in lines 88 and 89 assumes that `WIKI_EN_URI` is the base string for `WIKI_EN_TILE_URI` (which has a typo). You should reflect that in the creation of the strings.

This should be changed to 
```java
WIKI_EN_URI = "htp://rdf..../wikipedia.en";
WIKI_EN_TITLE_URI = WIKI_EN_URI + "_title";
```